src/mumble/ListenerLocalVolumeSlider.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2005-2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_LISTENERLOCALVOLUMESLIDER_H_
+#define MUMBLE_MUMBLE_LISTENERLOCALVOLUMESLIDER_H_
+
+
+class ClientUser;
+class Channel;
+
+#include <QMap>
+
+#include "ClientUser.h"
+#include "Channel.h"
+#include "ChannelListener.h"
+
+#include <VolumeSliderWidgetAction.h>
```

Comment:
```suggestion
#include "VolumeSliderWidgetAction.h"
```

We kinda follow the convention of using quotes for all of our own headers and <> for all external ones